common4j/build.gradle

```diff
@@ -187,10 +213,15 @@ dependencies {
     testAnnotationProcessor "org.projectlombok:lombok:$rootProject.ext.lombokVersion"
     testImplementation "junit:junit:$rootProject.ext.junitVersion"
     testImplementation "org.mockito:mockito-core:$rootProject.ext.mockitoCoreVersion"
+
+    testFixturesCompileOnly 'com.github.spotbugs:spotbugs-annotations:3.1.12'
+    testFixturesCompileOnly "org.projectlombok:lombok:$rootProject.ext.lombokVersion"
+    testFixturesAnnotationProcessor "org.projectlombok:lombok:$rootProject.ext.lombokVersion"
+    resolvableTestFixturesImplementation 'org.robolectric:junit:4.5.1'
 }
 
-sourceCompatibility = "1.7"
-targetCompatibility = "1.7"
+sourceCompatibility = "1.8"
```

Comment:
Why is this change needed?

Reply:
I think I needed it for some other code I wrote in the bigger PR. So this is a remnant from that but since it belonged in `build.gradle` therefore I included it here as all the `build.gradle` changes were here so it was easier for me to organize the PRs. 

Also I think we've actually already technically been on Java 8 since Melissa's YubyKit changes (YubyKit SDK is on Java 8). We just hadn't explicitly made the change in build.gradle here.